src/components/Chart/index.d.ts

```diff
@@ -17,6 +17,8 @@ export interface ChartProps extends BaseProps {
     disableXAxisTickLabels?: boolean;
     disableYAxisTickLabels?: boolean;
     maintainAspectRatio?: boolean;
+    plugins?: Array<any>;
+    datalabels?: object;
```

Comment:
the plugins feature is for any possible plugin, right? which means there will be more props depending on the plugins passed in `plugins` prop, I would say we need to find some dynamic manner of defining those props. 

i.e. a plugin named `foo` will receive it's config data through `foo` prop
```js
import foo from 'chart-plugin-foo';

<Chart plugins={{ foo }} foo={fooConfigAtChartLevel}>
    <Dataset foo={fooConfigAtDatasetLevel}>
</Chart>
```